tests/posix/threads_base/src/pthread.c

```diff
@@ -399,7 +396,7 @@ ZTEST(pthread, test_pthread_tryjoin)
 	zassert_ok(pthread_tryjoin_np(th, &retval));
 }
 
-ZTEST(pthread, test_pthread_timedjoin)
+ZTEST(posix_threads_base, test_pthread_timedjoin)
```

Comment:
This is technically nonportable. We should consider having a Kconfig to enable it

Reply:
Not a big deal though, we can fix that later.